pfcpiface/main.go

```diff
@@ -6,14 +6,14 @@ package main
 import (
 	"encoding/json"
 	"flag"
+	fqdn "github.com/Showmax/go-fqdn"
```

Comment:
this should go back down since its not standard library

Reply:
go format moved it up. I can pull it down again.

Reply:
oh ok I though in some of the formatters they dont rearrange things across import blocks separated by a blank line